physionet-django/console/views.py

```diff
@@ -506,12 +525,32 @@ def manage_published_project(request, project_slug):
             project.make_zip()
             messages.success(request, 'The zip of the main files has been generated.')
 
+        elif 'bucket' in request.POST:
+            slug = request.POST['bucket'].lower()
+            if not check_bucket(slug, project.version): # Check if the bucket exists
```

Comment:
These comments might be clearer if not inline (pep8 discourages inline comments, I think).